WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -96,10 +101,12 @@ private void StepConnectionConfirmationPhase()
 						Prison.Note(alice, round.Id);
 					}
 					round.Alices.RemoveAll(x => alicesDidntConfirm.Contains(x));
```

Comment:
`round.Alices.RemoveAll(x => alicesDidntConfirm.Contains(x));` returns to you how many elements were removed from `round.Alices`. 

Your log may be misleading.

Reply:
Done.